core/lib/lh-error.js

```diff
@@ -158,26 +159,34 @@ class LighthouseError extends Error {
     if (err instanceof LighthouseError) {
       // Remove class props so that remaining values were what was passed in as `properties`.
       // eslint-disable-next-line no-unused-vars
-      const {name, code, message, friendlyMessage, lhrRuntimeError, stack, ...properties} = err;
+      const {name, code, message, friendlyMessage, lhrRuntimeError, stack, cause, ...properties} = err;
+      const serializedCause = cause ?
+        LighthouseError.stringifyReplacer(/** @type {Error} */ (cause)) :
```

Comment:
this (and the reviver) shouldn't need to go recursive, JSON.stringify/parse should step through the properties so it'll work as expected/hoped